compiler/modules/front/ast.bal

```diff
@@ -113,6 +113,10 @@ type BinaryRelationalOp "<" | ">" | "<=" | ">=";
 type BinaryEqualityOp  "==" | "!=" | "===" | "!==";
 type RangeOp  "..." | "..<";
 
+type CompAssnArithmeticOp "+=" | "-=" | "/=" | "*=";
```

Comment:
Elsewhere the code is using "Assign" as the abbreviation for "Assignment": please stick with that.

Better not to abbreviate "Compound", I think.